documentation/API_documentation/QoD_Stable_Bandwidth_API.md

```diff
@@ -0,0 +1,180 @@
+# Overview
+
+<span class="colour" style="color:rgb(0, 0, 0)">The Quality-On-Demand (QoD) API for stable throughput provides programmable interface for developers and other users (capabilities consumers) to specify stable data-transfer throughput managed by Telco networks without necessity to have in-depth knowledge of the 4G/5G system complexity by abstracting the internal complexity of telecom systems [1].</span>
+
+## 1\. Introduction
+
+There are many real-world use cases which could exploit the potential of programmable networks. Several of these use case implementations are still work in progress or have not been operationalized as developers have no means to configure or request stable latency or bandwidth during congestion from a mobile network operator which is critical. Use cases from industrial (IoT), VR/Gaming, broadcasting, autonomous driving and many others fall in such a category since they demand network communication quality and are sensitive to any change in transmission conditions. 
+
+The QoD throughput API offers the application developers and users the capability to request for stable throughput for a specified App-Flow(s) between application clients and backend services (Application Servers). The developer has a pre-defined set of QoS_Profiles which he could choose from depending on his throughput requirements. 
```

Comment:
"The QoD throughput API offers the application developers and users the capability to request for stable throughput for a specified App-Flow(s) between application clients and backend services (Application Servers)." -> "The QoD throughput API offers to application developers the capability to request stable throughput for a specified Application Flow between User Equipment and Application Server."

This should not be in plural (flows, clients, servers) if API creation does one flow, and needs to be called more times for more flows.
Also trying to be more precise on between which exact locations will API improve the throughput.

Reply:
Sentence changed in [1fd0cfe](https://github.com/camaraproject/QualityOnDemand/pull/17/commits/1fd0cfe1c718e6362cfb0f1536790f28fb003d8e) (see also comment in Latency API doc).